scripts/metavar.py

```diff
@@ -1025,7 +1027,7 @@ def conditional(self, vdicts):
                     vars_needed.append(dvar)
         return (conditional, vars_needed)
 
-    def write_def(self, outfile, indent, wdict, allocatable=False,
+    def write_def(self, outfile, indent, wdict, allocatable=False, target=None,
```

Comment:
would it more consistent to have target default to False instead of None?

Reply:
Good catch. Done